### PR TITLE
Data cookbooks update 4

### PR DESCRIPTION
**Pull request by Jkovarik**

### CODE REVIEW

File: docs/data-cookbooks/sips-workflow.md [hunk 1]
```diff
@@ -97,41 +97,53 @@ Finally, let's create a [rule](data-cookbooks/setup.md#rules). In this example w
...
+1. ParsePdr - [source](https://github.com/nasa/cumulus/tree/master/tasks/parse-pdr)
+2. QueueGranules - [source](https://github.com/nasa/cumulus/tree/master/tasks/queue-granules)
+3. CheckStatus - [source](https://github.com/nasa/cumulus/tree/master/tasks/pdr-status-check)
 
 ![Screenshot of execution graph for SIPS Parse PDR workflow in AWS Step Functions console](assets/sips-parse-pdr.png)
 
-_Example configuration for this workflow can be found in the `ParsePdr` object defined in Cumulus core's [example](https://github.com/nasa/cumulus/blob/master/example/workflows/sips.yml)_
+_An example workflow module configuration can be viewed in the Cumulus source for the [parse_pdr_workflow](https://github.com/nasa/cumulus/blob/master/example/cumulus-tf/parse_pdr_workflow.tf)._
+
+_**Please note:** To use this example workflow module as a template for a new workflow in your deployment the `source` key for the workflow module would need to point to a release of the cumulus-workflow (terraform-aws-cumulus-workflow.zip) module on our [release](https://github.com/nasa/cumulus/releases) page, as all of the provided Cumulus workflows are internally self-referential._
```
Comment (markdboyd) [inline docs/data-cookbooks/sips-workflow.md]: Definitely worth highlighting. Wonder if there's any way we can just say this once? Maybe a footnote or something else?

File: docs/data-cookbooks/sips-workflow.md [hunk 2]
```diff
@@ -97,41 +97,53 @@ Finally, let's create a [rule](data-cookbooks/setup.md#rules). In this example w
...
+
+1. SyncGranule - [source](https://github.com/nasa/cumulus/tree/master/tasks/sync-granule).
+2. CmrStep - [source](https://github.com/nasa/cumulus/tree/master/tasks/post-to-cmr)
 
-**Note:** Hitting CmrStep is not required and can be left out of the processing trajectory if desired (for example, in testing situations).
+Additionally this workflow requires a ProcessingStep you must provide. The "AsterProcess," "ModisProcess," and "LegacyProcess" steps in the workflow picture below are examples of custom processing steps.
+
+**Note:** Using the CmrStep is not required and can be left out of the processing trajectory if desired (for example, in testing situations).
 
 ![Screenshot of execution graph for SIPS IngestGranule workflow in AWS Step Functions console](assets/sips-ingest-granule.png)
```
Comment (markdboyd) [inline docs/data-cookbooks/sips-workflow.md]: I'm betting most of those diagrams are inaccurate, esp bc they include Report/StopStatus steps. We should circle back in another PR to update them all

Comment (Jkovarik) [reply docs/data-cookbooks/sips-workflow.md]: Good point -  might as well do it now.  I can issue a separate PR though.

File: docs/data-cookbooks/sips-workflow.md (hunk 1 above)
Comment (Jkovarik) [reply docs/data-cookbooks/sips-workflow.md]: I thought about that, but this *particular* note feels like we need to make the point of making a point.           That's just my opinion though, I'm willing to make it less pointed.

Comment (markdboyd) [reply docs/data-cookbooks/sips-workflow.md]: I'm good with the repetition for the sake of clarity

**Pull request merged**